src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -282,220 +283,206 @@ namespace CondenserLoopTowers {
                                                      state.dataIPShortCut->cAlphaFieldNames(1),
                                                      ErrorsFound);
-            state.dataCondenserLoopTowers->towers(TowerNum).Name = AlphArray(1);
-            state.dataCondenserLoopTowers->towers(TowerNum).thisTowerNum = TowerNum;
-            state.dataCondenserLoopTowers->towers(TowerNum).TowerType = DataPlant::PlantEquipmentType::CoolingTower_SingleSpd;
-            state.dataCondenserLoopTowers->towers(TowerNum).TowerMassFlowRateMultiplier = 2.5;
-            state.dataCondenserLoopTowers->towers(TowerNum).WaterInletNodeNum =
-                NodeInputManager::GetOnlySingleNode(state,
-                                                    AlphArray(2),
-                                                    ErrorsFound,
-                                                    DataLoopNode::ConnectionObjectType::CoolingTowerSingleSpeed,
-                                                    AlphArray(1),
-                                                    DataLoopNode::NodeFluidType::Water,
-                                                    DataLoopNode::ConnectionType::Inlet,
-                                                    NodeInputManager::CompFluidStream::Primary,
-                                                    DataLoopNode::ObjectIsNotParent);
-            state.dataCondenserLoopTowers->towers(TowerNum).WaterOutletNodeNum =
-                NodeInputManager::GetOnlySingleNode(state,
-                                                    AlphArray(3),
-                                                    ErrorsFound,
-                                                    DataLoopNode::ConnectionObjectType::CoolingTowerSingleSpeed,
-                                                    AlphArray(1),
-                                                    DataLoopNode::NodeFluidType::Water,
-                                                    DataLoopNode::ConnectionType::Outlet,
-                                                    NodeInputManager::CompFluidStream::Primary,
-                                                    DataLoopNode::ObjectIsNotParent);
+            auto &tower = state.dataCondenserLoopTowers->towers(TowerNum);
+            tower.Name = AlphArray(1);
+            tower.thisTowerNum = TowerNum;
+            tower.TowerType = DataPlant::PlantEquipmentType::CoolingTower_SingleSpd;
+            tower.TowerMassFlowRateMultiplier = 2.5;
+            tower.WaterInletNodeNum = NodeInputManager::GetOnlySingleNode(state,
+                                                                          AlphArray(2),
+                                                                          ErrorsFound,
+                                                                          DataLoopNode::ConnectionObjectType::CoolingTowerSingleSpeed,
+                                                                          AlphArray(1),
```

Comment:
Use `tower.Name` here rather than `AlphArray(1);`. If you've already captured a field in the C++ object, use the C++ object field instead.